goon/code/datums/browserOutput.dm

```diff
@@ -70,7 +70,7 @@ GLOBAL_LIST_EMPTY(bicon_cache)
 		for(var/asset in GLOB.chatResources)
 			owner << browse_rsc(wrap_file(asset))
 
-		for(var/subattempts in 1 to 3)
+		for(var/subattempts in 1 to 3) // GDN note- Remove this later it DOES NOT WORK
```

Comment:
Why not just remove this now?

Reply:
Not really sure about leaving personal notes in the files but as long as you come back to remove this then it's whatever

Reply:
Ye I was gonna do it after this pr got merged lol